src/org/traccar/reports/Summary.java

```diff
@@ -90,7 +91,10 @@ public static void getExcel(OutputStream outputStream,
             jxlsContext.putVar("to", to);
             jxlsContext.putVar("distanceUnit", ReportUtils.getDistanceUnit(userId));
             jxlsContext.putVar("speedUnit", ReportUtils.getSpeedUnit(userId));
-            jxlsContext.putVar("timezone", from.getZone());
+            jxlsContext.putVar("webUrl", Context.getVelocityEngine().getProperty("web.url"));
+            jxlsContext.putVar("dateTool", new DateTool());
+            jxlsContext.putVar("timezone", ReportUtils.getTimezone(userId));
+            jxlsContext.putVar("locale", Locale.getDefault());
```

Comment:
Is it possible to share context initialization code? It keeps growing, so it would be nice to remove some code duplication.